konch.py

```diff
@@ -613,13 +614,54 @@ def start(self) -> None:
         return None
 
 
+class BPythonCursesShell(Shell):
+    """The BPython Curses shell."""
+
+    def _embed(self, **kwargs) -> None:
```

Comment:
I tried to keep it consistent with what `embed` call in `BPythonShell` inside `start` does. I too agree that code can be simplified but I still it in this way so that any changes done in future for `BPythonShell` can almost be copied down to `BPythonCursesShell` without thinking about missing similar calls in `start` like `embed`.